src/main/viz/VariantTrack.js

```diff
@@ -126,7 +126,7 @@ class VariantTrack extends React.Component {
     if (variants && variants.length>0) {
       var data = [];
       for (var i=0;i<variants.length;i++) {
-        data.push({id: variants[i].id,vcfLine:variants[i].vcfLine});
+        data.push({id:variants[i].id,vcfLine:variants[i].vcfLine,ref:variants[i].ref,alt:variants[i].alt});
```

Comment:
sorry for the styling inconsistency here; if you had a chance, would you mind fixing the style here? OK if you don't get to it:

```javascript
// ...

for (var i=0; i < variants.length; i++) {
  data.push({
    id: variants[i].id,
    vcfLine: variants[i].vcfLine,
    ref: variants[i].ref,
    alt: variants[i].alt
  });
}

// ...
```